src/config.js

```diff
@@ -137,6 +137,19 @@ function extractRegex(envVar) {
   return undefined;
 }
 
+function parseParagonThemeUrls(paragonUrlsJson) {
```

Comment:
[optional/question]: Since we are not using TypeScript in this file, does it make sense to add [JSDocs](https://jsdoc.app/about-getting-started) instead of comments that will contain both the description of the function and some types for `paragonUrlsJson`? I think this approach will make the code easier to read, better to understand, and will simplify the potential transition to TypeScript in the future.

This question applies to all new functions in this PR.

Reply:
I add the JSDoc for the mentioned function and enhance the other ones.

Reply:
Resolved